wen-restart/src/wen_restart.rs

```diff
@@ -451,6 +449,23 @@ mod tests {
         pub wen_restart_proto_path: PathBuf,
     }
 
+    fn insert_slots_into_blockstore(
+        blockstore: Arc<Blockstore>,
+        first_parent: Slot,
+        slots_to_insert: Vec<Slot>,
+    ) {
+        let mut parent = first_parent;
+        for slot in slots_to_insert {
+            let entries = entry::create_ticks(1, 0, Hash::default());
+            let shreds = blockstore::entries_to_test_shreds(
+                &entries, slot, parent, true, // is_full_slot
+                0, true, // merkle_variant
+            );
+            blockstore.insert_shreds(shreds, None, false).unwrap();
+            parent = slot;
+        }
+    }
+
```

Comment:
I believe we have a method `Blockstore::make_chaining_slot_entries()` that already does this

Reply:
Done. I needed to add first_parent there, otherwise the first one chains to 0.